sdks/python/apache_beam/runners/dataflow/dataflow_metrics.py

```diff
@@ -200,9 +220,18 @@ def _get_metrics_from_dataflow(self):
       self._cached_metrics = job_metrics
     return job_metrics
 
+  def all_metrics(self):
```

Comment:
Can we expand query() and __populate_metrics() with "query_user" and "query_v1b3" (based on code, can be query_system) parameters. This will keep only 1 method that will be more flexible than two of currently existing.

Default values should keep code backwards compatible.

Reply:
No, we need to discuss that kind of change on the dev list. This PR introduces he bare minimum to support integration testing. To enhance this API we need to design on the dev list.